packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -816,45 +836,41 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
         }
     }
 
-    protected _updateRotationSpheres() {
-        const rotateSpheres = this._rotateSpheresParent.getChildMeshes();
+    protected _updateRotationAnchors() {

```

Comment:
if this function is executed often, would be good to use reusable vectors instead of creating new ones when using `addInPlace`

Reply:
Good catch. It's simpler now.